SU-HSDO/stanford_mrc
# MRCD8-298 wrapped up remianing css to scss

# READY FOR REVIEW

# Summary
- Packaged up remaining "Remove Me" file CSS, added to appropriate SCSS files
- Left "Remove Me" file in the file system since we have temporarily added mobile menu items to that file on another branch; will remove later

# Needed By (Date)
- End of sprint (3/15)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Verify all CSS has been removed from "Remove Me" file and code is that same

# Affected Projects or Products
- MRC

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-298

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
